CHANGELOG.rst

```diff
@@ -2,6 +2,29 @@
 Changelog
 =========
 
+Version 16.0
+=============
+
+* Fixed Deneb Readout errors to closer resemble reality. Fidelities were reported as errors. `#125 <https://github.com/iqm-finland/iqm-client/pull/125>`_
```

Comment:
Is this link related?

Reply:
Yeah, cherry picked the changes from there.

Reply:
It links to iqm-client, which does not involve IQMFakeDeneb

Reply:
Oh, what?

Reply:
Find the right pr and added that.